templates/Dockerfile.j2

```diff
@@ -23,10 +29,8 @@ RUN groupadd -g 1000 elasticsearch && adduser -u 1000 -g 1000 -d /usr/share/elas
 
 WORKDIR /usr/share/elasticsearch
 
-# Download/extract defined ES version. busybox tar can't strip leading dir.
+# Download and extract defined ES version. busybox tar can't strip leading dir.
 RUN wget --progress=bar:force {{ url_root }}/{{ tarball }} && \
-    EXPECTED_SHA=$(wget -O - {{ url_root }}/{{ tarball }}.sha1) && \
```

Comment:
It would be great if we could keep this for the real releases and conditionally skip for snapshot artifacts.